README.md

```diff
@@ -126,36 +126,34 @@ We have list of frequent participants in our codebase.
 - Want to be on this list? Become frequent participant by contributing more, [come with us](https://open.spotify.com/track/5kTBiVnjq9xKmZL9dNs8zL?si=9fc60b8b87764969)!
 - [You can learn about our Contributors base](https://github.com/kodadot/nft-gallery/graphs/contributors)
 
-## Docker-🐳 
+## Docker-🐳
 If you just want to try out our KodaDot on Kusama and have a full local setup with a local node, we assume you have [docker](https://docs.docker.com/get-docker/) and docker-compose installed.
 
-Run Kodadot locally
-```bash
-docker-compose up -d --build
-```
+- ### First time setup
```

Comment:
```suggestion
### First time setup
```

Reply:
It's intended, there are two steps.

Reply:
Alright, just wanted to keep README consistency